Tests/MongoSwiftTests/ClientSessionTests.swift

```diff
@@ -26,7 +26,8 @@ final class ClientSessionTests: MongoSwiftTestCase {
         (name: "find", body: { _ = try $0.find([:], session: $1).nextOrError() }),
         (name: "aggregate", body: { _ = try $0.aggregate([], session: $1).nextOrError() }),
         (name: "distinct", body: { _ = try $0.distinct(fieldName: "x", session: $1) }),
-        (name: "count", body: { _ = try $0.count(session: $1) })
+        (name: "count", body: { _ = try $0.countDocuments(session: $1) }),
```

Comment:
would probably be best to use the full names here (i.e. "countDocuments" and "estimatedDocumentCount")